cmd/agent/subcommands/run/command.go

```diff
@@ -185,6 +186,7 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 				LogParams:            log.ForDaemon(command.LoggerName, "log_file", path.DefaultLogFile),
 			}),
 			fx.Supply(pidimpl.NewParams(cliParams.pidfilePath)),
+			logging.EnableFxLoggingOnDebug[log.Component](),
```

Comment:
Would be great to include `logging.EnableFxLoggingOnDebug` as part of `FxAgentBase`.

Reply:
I imagined